src/odemis/dataio/tiff.py

```diff
@@ -725,8 +770,13 @@ def _updateMDFromOME(root, das):
 
             try:
                 hex_str = che.attrib["Color"] # hex string
-                hex_str = hex_str[-8:] # almost copy of conversion.hex_to_rgb
-                tint = tuple(int(hex_str[i:i + 2], 16) for i in [0, 2, 4, 6])
+                try:
+                    # try to cast as int -> its uint32 stored, otherwise its hex str
+                    color_int = int(hex_str)
+                    tint = int32_to_rgba(color_int)
+                except Exception:
+                    hex_str = hex_str[-8:] # almost copy of conversion.hex_to_rgb
+                    tint = tuple(int(hex_str[i:i + 2], 16) for i in [0, 2, 4, 6])
```

Comment:
we can also use the function call here

Reply:
this is slightly different than what is in hex_to_rgb, and i'm not sure why so I don't want to change it at this stage